### PR TITLE
add wtInitStatus, mntAlgStatus and imuInitStatus from ESF-STATUS

### PR DESCRIPTION
**Pull request by andrejlevkovitch**
### Required for all PRs:

- [ ] Signed [CLA](https://influxdata.com/community/cla/).
- [ ] Associated README.md updated.
- [ ] Has appropriate unit tests.

@TPiowczykHay FYI: the fields can contains values:
- 0 - off
- 1 - initialization
- 2 - initialized

### CODE REVIEW

File: plugins/inputs/ublox/ublox.go [hunk 1]
```diff
@@ -158,6 +170,10 @@ func (s *UbloxDataCollector) Gather(acc telegraf.Accumulator) error {
 
 		if s.lastPos.FusionMode != None {
 			metrics["fusion_mode"] = s.lastPos.FusionMode
+
+			metrics["wtInitStatus"] = s.lastPos.WtInitStatus
+			metrics["mntAlgStatus"] = s.lastPos.MntAlgStatus
+			metrics["imuInitStatus"] = s.lastPos.ImuInitStatus
```
Comment (TPiowczykHay) [inline plugins/inputs/ublox/ublox.go:176]: Suggestion just for sake of keeping the same naming convention as for other metrics
```suggestion
			metrics["wt_init_status"] = s.lastPos.WtInitStatus
			metrics["mnt_alg_status"] = s.lastPos.MntAlgStatus
			metrics["imu_init_status"] = s.lastPos.ImuInitStatus
```

Comment (andrejlevkovitch) [reply plugins/inputs/ublox/ublox.go:176]: definitely not, as it only adds confusion - in documentation to ublox are used exactly this naming

Comment (andrejlevkovitch) [reply plugins/inputs/ublox/ublox.go:176]: also in the request @jerelbn used exactly this naming

Comment (andrejlevkovitch) [reply plugins/inputs/ublox/ublox.go:176]: and here we never had any naming convention, except: using the same names as in ublox doc

Comment (TPiowczykHay) [reply plugins/inputs/ublox/ublox.go:176]: 1. I thing you misunderstood what is convention - it creates "itself" when first code is written, there is no need for any documentation or force-formatting to make convention being convention
2. ublox documentation has also `fusionMode` instead `fusion_mode` we have, same for `fix_type` and all other several names
  a. this mean the convention in this repo was already set to `snake_case`
  b. `except: using the same names as in ublox doc` is literally a lie
3. It's much more confusing why would we have multiple naming conventions that is being introduced NOW by this PR

Comment (andrejlevkovitch) [reply plugins/inputs/ublox/ublox.go]: > except: using the same names as in ublox doc is literally a lie

hm, I'd say __mistake__, but yeah my mistake, changed it

**Review by TPiowczykHay — DISMISSED** [review]
Looks good to me, we could update README to not miss it, but we can do that on other PR as well

**Comment by andrejlevkovitch** [comment]
@TPiowczykHay updated README, please reapprove

**Review by TPiowczykHay — APPROVED** [review]

**Pull request merged**